4_Marketing-Support/support.md

```diff
@@ -42,11 +42,11 @@ Support related issues and topics are discussed on the product's support channel
 
 ## Live Chat Support
 
-Intercom's Messenger app is available on our salespages. This makes it easier for customers to contact us for issues, requests or sales questions.
+Help Scout Beacons are added on our sales pages and in-app. This makes it easier for customers to contact us for issues, requests or sales questions. Since we're supporting multiple projects, make sure to toggle the custom availability by mailbox on first log in.
```

Comment:
```suggestion
Help Scout Beacons are added on our sales pages and in-app. This makes it easier for customers to contact us for issues, requests or sales questions. Since we're supporting multiple projects, make sure to toggle your custom availability by mailbox on first login in the top right menu.
```